public/angular/templates/media/settings/elmo.html

```diff
@@ -0,0 +1,34 @@
+<div>
+  <form name="settingsForm" class="form-horizontal" novalidate>
+    <div class="modal-header">
+      <button class="close" ng-click="close()" aria-hidden="true">×</button>
+      <h4 class="modal-title">
+        <div>
+          <div class="icon-left">&nbsp;Edit ELMO Settings Source</div>
+      </h4>
+      </div>
+      <div class="modal-body">
+        <alert type="info" ng-show="success">The credentials are valid. You need to save them before continuing. </alert>
+        <alert type="info" ng-show="failure">The credentials are invalid or they have expired</alert>
```

Comment:
This message should have a period as does the success message. Please try to be consistent with such things.